src/SerializableAsync.ts

```diff
@@ -6,14 +6,21 @@ import { Serializable } from "./Serializable"
 import { SerializableBase } from "./SerializableBase"
 
 export class SerializableAsync extends SerializableBase implements ISerializable {
+    /**
+     * Parses the given object to the class defined in `@type`.
+     *
+     * @param value the object to deserialize
```

Comment:
@ param value The JSON object to parse. The object must have the "@ type" property set in order to be parsed, in addition to possible "@ context" and "@ version" properties.